drivers/bridge/setup_device.go

```diff
@@ -31,7 +32,29 @@ func setupDevice(config *NetworkConfiguration, i *bridgeInterface) error {
 	}
 
 	// Call out to netlink to create the device.
-	return netlink.LinkAdd(i.Link)
+	if err = netlink.LinkAdd(i.Link); err != nil {
+		return types.InternalErrorf("Failed to program bridge link: %s", err.Error())
+	}
+
+	if config.FixedCIDRv6 != nil {
```

Comment:
This is done in this function to retain the existing functionality.
If you look closer in this function, it is called ONLY for the case of bridge non-existence case.
If we move it to setupFixedCIDRv6, then we wont be able to retain that functionality.